pkgs/by-name/li/libfprint-focaltech-2808-a658/package.nix

```
@@ -0,0 +1,104 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  rpm,
+  cpio,
+  glib,
+  gusb,
+  pixman,
+  libgudev,
+  nss,
+  libfprint,
+  cairo,
+  pkg-config,
+  autoPatchelfHook,
+  makePkgconfigItem,
+  copyPkgconfigItems,
+}:
+
+# https://discourse.nixos.org/t/request-for-libfprint-port-for-2808-a658/55474
+let
+  # library name in source
```

Nit: It was not immediately clear what this meant. Perhaps something like this would be clearer:

```suggestion
  # The provided `.so`'s name in the binary package we fetch and unpack
```